python/src/CMakeLists.txt

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+pyvelox_add_module(
+  legacy
+  MODULE
+  legacy/complex.cpp
+  legacy/conversion.cpp
+  legacy/pyvelox.cpp
+  legacy/serde.cpp
+  legacy/signatures.cpp)
+
+target_link_libraries(
+  legacy
+  PRIVATE
+    velox_type
+    velox_vector
+    velox_core
+    velox_exec
+    velox_parse_parser
+    velox_functions_prestosql
+    velox_functions_spark
+    velox_aggregates
+    velox_functions_spark_aggregates)
+
+if(VELOX_PYTHON_LEGACY_ONLY)
```

Comment:
Is it possible to print out what is dependent on this so we can flag PYTHON_LEGACY_ONLY at the top level ?

Reply:
I want to reorganize the options soon, it's getting pretty unwildy